### PR TITLE
Swift Package Manager

### PR DESCRIPTION
**Pull request by maxxfrazer**
- Added Package.swift with same minimum versions etc as pod spec

I moved ShowTime.swift into Sources/ShowTime because SwiftPM doesn't like when it's at the top level of a repo with non-swift files. If needed I can move it to be in ./ShowTime or ./ShowTime/Classes, the Package.swift and podspec will just need to be updated to reflect that.

### CODE REVIEW

**Comment by maxxfrazer** [comment]
Also I've set the target to develop, unsure if you prefer master or develop for PRs

**Comment by KaneCheshire** [comment]
Also for some reason CI isn't being reported even when it passes

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,6 @@
 /Example/ShowTime.xcworkspace/xcuserdata/kanecheshire.xcuserdatad/UserInterfaceState.xcuserstate
```
[the remark this answers is not in the document]
Comment (maxxfrazer) [reply .gitignore]: I've changed it a little now in my latest commit

[the remark this answers is not in the document]
Comment (maxxfrazer) [reply .gitignore]: > Also for some reason CI isn't being reported even when it passes

I've added this forked repo to my travis account - should be running now!

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,7 @@
-/ShowTime.xcodeproj/xcuserdata/kanecheshire.xcuserdatad/xcdebugger
-/Example/ShowTime.xcworkspace/xcuserdata/kanecheshire.xcuserdatad/UserInterfaceState.xcuserstate
```
Comment (KaneCheshire) [inline .gitignore:2]: Thank youuuu

**Comment by maxxfrazer** [comment]
Ok looks to all be good now on travis to me, just doesn't show in the PR, I've added a screenshot to confirm though.

Screen Shot 2020-04-07 at 16 39 13

**Comment by KaneCheshire** [comment]
Yeah really not sure what's up with Travis on this repo!

**Comment by KaneCheshire** [comment]
Happy to merge this?

**Comment by maxxfrazer** [comment]
Yep, LGTM!

**Pull request merged**